python/core/auto_generated/processing/qgsprocessingparametertininputlayers.sip.in

```diff
@@ -0,0 +1,99 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/core/processing/qgsprocessingparametertininputlayers.h           *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+class QgsProcessingParameterTinInputLayers: QgsProcessingParameterDefinition
+{
+%Docstring
+A parameter for processing algorithms that need a list of input vector layers to construct a TIN
+A valid value for this parameter is a list (QVariantList), where each item is a map (QVariantMap) in this form:
+{
+'Id':  string hat represents the unique Id of the vector layer,
```

Comment:
```suggestion
'Id':  string that represents the unique Id of the vector layer,
```

Reply:
Other similar parameters (e.g. QgsProcessingParameterVectorTileWriterLayers) use camel case keys with lowercase first letter -- would be good to maintain that convention here